wrappers/PhyML/phyml/tool_dependencies.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<tool_dependency>
+	<package name="phyml" version="3.1">
+		<repository name="package_phyml_3_1" owner="galaxyuser" toolshed="http://onottar669810x.agr.gc.ca:9009" changeset_revision="03e72ec051ba" />
```

Comment:
Remove the changeset_revision - it is not necessary to install the tool, while keeping it there creates an extra task for the sys admin to modify.